Framework/GenAIFramework/Code/Source/O3DESceneGenerationAssistantPythonFeature/O3DESceneGenerationAssistantPythonFeature.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <GenAIFramework/Feature/FeatureBase.h>
+#include <GenAIFramework/Feature/PythonFeatureBase.h>
+#include <GenAIFramework/GenAIFrameworkTypeIds.h>
+
+#include <AzCore/RTTI/RTTI.h>
+#include <AzCore/Serialization/SerializeContext.h>
+
+namespace GenAIFramework
+{
+    class O3DESceneGenerationAssistantPythonFeature : public PythonFeatureBase
+    {
+    public:
+        AZ_RTTI(O3DESceneGenerationAssistantPythonFeature, "{7c524a1b-a175-4c26-beb1-dbefb7f13f50}", PythonFeatureBase);
+        AZ_CLASS_ALLOCATOR(O3DESceneGenerationAssistantPythonFeature, AZ::SystemAllocator, 0);
+
+        O3DESceneGenerationAssistantPythonFeature() = default;
+        O3DESceneGenerationAssistantPythonFeature(AZ::u64 agentId, AZ::u64 conversationId);
+        ~O3DESceneGenerationAssistantPythonFeature() = default;
+
+        static void Reflect(AZ::ReflectContext* context);
+
+    private:
+        constexpr static const char* PythonSceneGenerationAssistantScript = "Editor/Scripts/scene_generation/scene_generation_assistant.py";
+
+        AZStd::string m_sceneGenerationAssistantPythonScriptLocation;
```

Comment:
Please keep the names shorter: this is a member of a class called _SceneGenerationAssistant_, no need to repeat it.

Reply:
I was following the convention from o3de assistant example (and went too far, but if we simplify it here then let's also simplify it there)

Reply:
Applied [8976295](https://github.com/RobotecAI/o3de-genai-gems/pull/158/commits/89762950bc0059d16d7fc100f1f8ee4e8275401c). Also names in O3DE Assistant (Python) have been adapted.